### ISSUE TITLE
[Feature request] Logging support

### ISSUE THREAD

**Issue by fleroviux** [nba-emu/NanoBoyAdvance#248]
Emulators like mGBA and VBA allow logging from GBA code into a debugger/log window via special hardware behaviour.
This is useful for developing GBA homebrew. We may want to implement mGBA's logging interface for interoperability.

This was requested by GValiente on the GBADev Discord server.

**Comment by fleroviux** [comment]
Step 1 was completed in https://github.com/nba-emu/NanoBoyAdvance/commit/f85c7ed088b7323e6174fb9f481fb8b81c54661c. Now we just need a nice UI to display and filter the logs.

**Comment by GValiente** [comment]
Thanks for the feature!

It seems NBA takes too much time in flushing (showing printed lines), unlike mGBA, which flushes after each new line.

**Comment by fleroviux** [comment]
@GValiente in the latest dev build both of these issues should be fixed now.

**Comment by GValiente** [comment]
> @GValiente in the latest dev build both of these issues should be fixed now.

Are there public dev builds available?

**Comment by fleroviux** [comment]
> > @GValiente in the latest dev build both of these issues should be fixed now.
> 
> Are there public dev builds available?

Yes, you can download them either from https://nightly.link/nba-emu/NanoBoyAdvance/workflows/build/master or https://github.com/nba-emu/NanoBoyAdvance/actions.

**Comment by GValiente** [comment]
It works! Thanks.
